src/content/ai/genui/get-started.md

```diff
@@ -0,0 +1,473 @@
+--- 
+title: Get started with the GenUI SDK for Flutter
+breadcrumb: Get started
+description: >-
+  Learn how to use GenUI SDK for Flutter to add it
+  to your existing Flutter app.
+---
+
+This guide explains how to get started with the
+[`genui`][] package. The SDK's key components
+are described in the [Key components][] page.
+
+:::experimental
+The `genui` package is in
+alpha and is likely to change.
+:::
+
+Use the following instructions to add `genui` to your Flutter app.
+The code examples show how to perform the instructions on a brand new
+app created by running `flutter create`, but you can follow the same
+steps for your existing Flutter app.
+
+[`genui`]: {{site.pub-pkg}}/genui
+[Key components]: /ai/genui/components
+
+## Configure your agent provider
+
+`genui` can connect to a variety of agent providers.
+Choose the section for your preferred provider.
+
+### Configure Firebase AI logic
+
+To use the built-in `FirebaseAiContentGenerator` to connect
+to Gemini using the Firebase AI Logic, follow these instructions:
+
+ 1. [Create a new Firebase project][] using the Firebase Console.
+
+ 2. [Enable the Gemini API][] for that project.
+
+ 3. Follow the first three steps in [Firebase's Flutter setup guide][]
+    to add Firebase to your app.
+
+ 4. In `pubspec.yaml`, add `genui` and `genui_firebase_ai`
+    to the `dependencies` section. As of this writing,
+    it's best to use pub's git dependency to
+    refer directly to this project's source.
```

Comment:
Do we need this anymore? The following pubspec snippet shows depending on them from pub.dev.

Are we able to suggest just running `dart pub add genui genui_firebase_ai`?